tests/Contacts/PhoneNumberTest.m

```diff
@@ -5,10 +5,29 @@
 #import <XCTest/XCTest.h>
 #import "PhoneNumber.h"
 
+@interface NSArray (PhoneNumberTest)
+
+@end
+
+#pragma mark -
+
+@implementation NSArray (PhoneNumberTest)
+
+- (id)test_firstObject
```

Comment:
Am I missing something or is this exactly the same as the built in `[myArray firstObject]`